changelog/6743.deprecation.rst

```diff
@@ -0,0 +1 @@
+Using `~sunpy.map.header_helper.meta_keywords()` is deprecated.
```

Comment:
```suggestion
Using `~sunpy.map.header_helper.meta_keywords` is deprecated.
Please see :ref:`Meta Keywords Table` for the list of metadata keywords used by `~sunpy.map.Map`.
```

Reply:
aahh thanks a lot there, i was having an unbelievably hard time figuring out how to do this (and the tests) :sob: 
was overthinking it, this looks very obvious in hindsight!

Reply:
We will need to see if this works first.

Reply:
it works! 🥳